src/components/shared/node_count_selector.js

```diff
@@ -114,7 +114,7 @@ class NodeCountSelector extends React.Component {
                 </SpanWrapper>
                 <NumberPicker
                   label=''
-                  max={99}
+                  max={999}
```

Comment:
Same with this, can we extract it to a constant somewhere? (for example, at the beginning of the file?)